docs/specs/escape.yml

```diff
@@ -135,6 +134,34 @@ inputs:
     Link to @a&b?displayProperty=description
     Link to @a&c
     Link to <xref:a%2A>
+  _themes/ContentTemplate/schemas/TestData.schema.json: |
+    {
+      "type": "object",
```

Comment:
remove unnecessary properties in this schema json to make the test case short and more readable. E.g., this case only needs:

```
{
    "properties": {
         "uid": { "contentType": "uid" }
    },
    "xrefProperties": ["name", "fullName", "description"]
}
```

Reply:
seems not working, i'm checking the diff between expected and actual

Reply:
@yufeih Updated, could you please help review? Let me know if I should do this for all `TestData.schema.json`

Reply:
Yep, do this for all other cases

Reply:
Done. Could you please help rewiew?